src/test/java/shop/RealItemTest.java

```diff
@@ -0,0 +1,34 @@
+package shop;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class RealItemTest {
+
+    private RealItem car;
+    @BeforeClass
+    public void setup() {
+        car = new RealItem();
+    }
+
+    @Test (description = "Real Item test - To String for Real Item")
+    public void realItemToStringTest() {
+        double realItemWeight = 123.0;
+        String realItemName = "testRealName";
+        double realItemPrice = 321.0;
+        car.setName(realItemName);
+        car.setPrice(realItemPrice);
+        car.setWeight(realItemWeight);
+        String expectedString = "Class: " + car.getClass() + "; Name: " + realItemName + "; Price: " + realItemPrice +"; Weight: " +realItemWeight;
```

Comment:
Better to use String.format in long string concatenation. Easier to read and maintain